SU2_DEF/src/SU2_DEF.cpp

```diff
@@ -1,319 +1,329 @@
-/*!
```

Comment:
Can you have a look what happened here ? Apparently git thinks the whole file has changed.